judge/contest_format/icpc.py

```diff
@@ -0,0 +1,114 @@
+from datetime import timedelta
+
+from django.core.exceptions import ValidationError
+from django.db import connection
+from django.template.defaultfilters import floatformat
+from django.urls import reverse
+from django.utils.html import format_html
+from django.utils.safestring import mark_safe
+from django.utils.translation import gettext_lazy
+
+from judge.contest_format.default import DefaultContestFormat
+from judge.contest_format.registry import register_contest_format
+from judge.timezone import from_database_time
+from judge.utils.timedelta import nice_repr
+
+
+@register_contest_format('icpc')
+class ICPCContestFormat(DefaultContestFormat):
```

Comment:
Should probably import `ICPCContestFormat` in `judge/contest_format/__init__.py`. Otherwise, it doesn't seem to show up in the contest admin.